client/src/landingPageContent/JoinUs.js

```diff
@@ -20,10 +22,63 @@ export default function JoinUs (props) {
             })
     }
 
+    const fetchSignUps = () => {
+        setSendingRequest(true)
+        return makeCall({}, `/sampleSignUps`, 'get')
+            .then(sampleSignUpsResponse => {
+                if (sampleSignUpsResponse) {
+                    setSampleSignUps(sampleSignUpsResponse.sampleSignUps)
+                    setSendingRequest(false)
+                } else {
+                    // TODO: error
+                    // Add error toast when we introduce toasts to system
+                    setSendingRequest(false)
+                }
+            })
+    }
+    const sampleSignUpsDisplayCards = () => {
+        return sampleSignUps && sampleSignUps.length ? 
+            <Segment>
+                <Header as='h4'>Get a chance to meet mentors like...</Header>
+                <Grid
+                    stackable
+                    textAlign='center'
+                >
+                    {
+                        sampleSignUps.map(alumnus => {
+                            let transitionDuration = [1000, 2000, 3000][Math.floor(Math.random() * [1000, 2000, 3000].length)]
+                            return (
```

Comment:
It feels like these cards could deserve their own component

Reply:
I don't think these cards will get reused anywhere else so I don't think that's worth the effort.